lib/pester.rb

```diff
@@ -71,6 +71,7 @@ def self.retry_action(opts = {}, &block)
         end
       end
     end
+    nil
```

Comment:
"sometimes i like to put an empty line when there's a return value after a big block but ymmv"

makes sense. kind of surprised rubocop didn't have an opinion either way